guava/src/com/google/common/collect/AbstractIterator.java

```diff
@@ -28,7 +28,7 @@
  * interface easier to implement for certain types of data sources.
  *
  * <p>{@code Iterator} requires its implementations to support querying the end-of-data status
- * without changing the iterator's state, using the {@link #hasNext} method. But many data sources,
+ * without changing the iterators state, using the {@link #hasNext} method. But many data sources,
```

Comment:
This is just plain wrong. If you argue it‘s _iterators_ in plural it should be _iterators'_ but I don‘t see any argument for that